[ingester] Optionally delay circuit breaker activation until after traffic starts (#9842)
grafana/mimir: docs/sources/mimir/configure/configuration-parameters/index.md

  # CLI flag: -ingester.push-circuit-breaker.cooldown-period
  [cooldown_period: <duration> | default = 10s]

>   # (experimental) How long after an initial request an activated circuit
>   # breaker should wait before becoming effectively active. During this time
>   # neither failures nor successes will not be counted.

Reviewer: ```suggestion
  # (experimental) Duration, in seconds, after an initial request an activated circuit
  # breaker should wait before becoming effectively active. During this time,
  # neither failures nor successes are counted.
```

Author: addressed in https://github.com/grafana/mimir/pull/9842/commits/24a3b0db894298d97e53aab419553c6d71074ec9

  # CLI flag: -ingester.read-circuit-breaker.cooldown-period
  [cooldown_period: <duration> | default = 10s]

  # (experimental) How long after an initial request an activated circuit
  # breaker should wait before becoming effectively active. During this time
  # neither failures nor successes will not be counted.

>   # CLI flag: -ingester.read-circuit-breaker.initial-delay

Reviewer: Same suggestion as above here.

Author: addressed in https://github.com/grafana/mimir/pull/9842/commits/24a3b0db894298d97e53aab419553c6d71074ec9
